pycrostates/cluster/_base.py

```diff
@@ -270,6 +270,11 @@ def reorder_clusters(self, mapping: dict = None,
         # re-order
         self._cluster_centers_ = self._cluster_centers_[order]
         self._clusters_names = [self._clusters_names[k] for k in order]
+        new_labels = np.zeros(self._labels_.shape)
```

Comment:
Should be in kmeans.py since it's an attribute from kmeans class only.
```
def reorder_clusters(...):
    super().reorder_clusters()
    # additional steps
    ...
```